pkg/api/steve/supportconfigs/supportconfigs.go

```diff
@@ -0,0 +1,57 @@
+package supportconfigs
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/rancher/rancher/pkg/version"
+)
+
+// supportConfig represents the serializable struct of support related information common to all rancher setups
+type supportConfig struct {
+	// SupportEligible - if this rancher product is eligible for support.
+	SupportEligible bool `json:"support_eligible,omitempty"`
```

Comment:
Change to camelcase if possible as I believe that's what we usually use for json fields.

Reply:
I'll need to double check the requirements to see if I can change this, but that's a good callout. I will double check and see if we can change this.

Reply:
@rmweir 

Took a look at this. I don't have any specific documentation on the names of these fields. However, I did not name these fields - this was taken from earlier work that Ryan did. I would prefer not to change this given that there is going to be a program which validates the supportconfig files produced by another area - I'm assuming that Ryan had more information then I do and put this here for a reason. 

In short - the available info that I have indicates that we need a specific schema and does not indicate camel case. I would prefer leave as is rather than making a somewhat risky change in order to meet our naming conventions. Please let me know if you disagree.